internal/query/current_sequence.go

```diff
@@ -66,34 +74,57 @@ func (q *Queries) ClearCurrentSequence(ctx context.Context, projectionName strin
 	if err != nil {
 		return errors.ThrowInternal(err, "QUERY-9iOpr", "Errors.RemoveFailed")
 	}
-	row := tx.QueryRow("select type from [show tables from zitadel.projections] where table_name not in ('locks', 'current_sequences', 'failed_events') and concat('zitadel.projections.', table_name) = $1;", projectionName)
-	var tableType string
-	if err := row.Scan(&tableType); err != nil || tableType == "" {
+	projectionQuery, args, err := sq.Select("count(*)").
```

Comment:
err not handled

Reply:
wouldn't it be easier to just check the length of `tables` variable? => one query instead of two

Reply:
problem is that we want to truncate `projectionName%` but `projectionName` must exist
otherwise you can pass `a` as projectionName and we will truncate / reset apps, authnKeys, ...